test/cases/resolving/commomjs-local-module/index.js

```diff
@@ -0,0 +1,23 @@
+var should = require("should");
+
+define("regular", function(require, exports, module) {
+  module.exports = "regular-module";
+});
+
+define("override-exports", function(require, exports, module) {
+  exports = "this one overrides exports reference";
```

Comment:
It doesn't, but we should return empty object and not undefined as it was before.